src/StateBridge.sol

```diff
@@ -45,6 +42,14 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
         address indexed previousOwner, address indexed newOwner, bool isLocal
     );
 
+    /// @notice Emmitted when the the StateBridge sets the root history expiry for OpWorldID (on Optimism)
+    /// @param rootHistoryExpiry The new root history expiry for OpWorldID
+    event SetRootHistoryExpiryOptimism(uint256 rootHistoryExpiry);
+
+    /// @notice Emmitted when the the StateBridge sets the root history expiry for PolygonWorldID (on Polygon)
+    /// @param rootHistoryExpiry The new root history expiry for PolygonWorldID
+    event SetRootHistoryExpiryPolygon(uint256 rootHistoryExpiry);
+
```

Comment:
There's no point having two separate events as we only ever do both at once. Just have an `event RootHistoryExpirySet(uint256 newExpiryTime)` instead and fire it once both are done.

Reply:
ok